src/ui/server/helpers/mappings/map-name-fields/index.ts

```diff
@@ -16,19 +15,7 @@ const {
  * @returns {Object} Application with name field characters
  */
 const mapNameFields = (application: Application): Application => {
-  const { business, buyer } = application;
-
-  if (business && business.businessContactDetail[FIRST_NAME]) {
-    const fieldValue = business.businessContactDetail[FIRST_NAME];
-
-    business.businessContactDetail[FIRST_NAME] = replaceCharacterCodesWithCharacters(fieldValue);
-  }
-
-  if (business?.businessContactDetail[LAST_NAME]) {
-    const fieldValue = business.businessContactDetail[LAST_NAME];
-
-    business.businessContactDetail[LAST_NAME] = replaceCharacterCodesWithCharacters(fieldValue);
-  }
```

Comment:
Seems we need to update this to include `policyContact` first/last name and add an E2E test like [this](https://github.com/UK-Export-Finance/exip/blob/main-application/e2e-tests/insurance/cypress/e2e/journeys/submit-name-fields-with-special-characters/submit-name-fields-with-special-characters-header-and-page-fields.spec.js)

Not a blocker. We can create a ticket and do next if it's not covered already in jira?